src/ALZ/Assets/alz-bicep-config/v0.14.1-pre.config.json

```diff
@@ -23,12 +23,12 @@
             "destination": "config/custom-parameters/managementGroups.parameters.all.json"
         },
         {
-            "source": "infra-as-code/bicep/modules/mgDiagSettings/parameters/mgDiagSettings.parameters.all.json",
-            "destination": "config/custom-parameters/mgDiagSettings.parameters.all.json"
+            "source": "infra-as-code/bicep/orchestration/mgDiagSettingsAll/parameters/mgDiagSettingsAll.parameters.min.json",
```

Comment:
There are a few values in here we could do with creating Computed substitutions and replacements for.   I can help with this if you'd like.